### PR TITLE
Add MoonJS

### PR DESCRIPTION
**Pull request by ctl**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -10221,6 +10221,14 @@
 			"icon": "xui.png",
 			"script": "[^a-z]xui.*\\.js",
 			"website": "http://xuijs.com"
+		},
+		"Moon": {
+			"cats": [
+				"12"
+			],
+			"icon": "moon.png",
+			"script": "moon(?:\\.min)?\\.js",
```
Comment (jvoisin) [inline src/apps.json]: Please prefix it with a `/` to reduce the number of false-positives.

Comment (ctl) [inline src/apps.json]: Now it's ok?

File: src/apps.json [hunk 2]
```diff
@@ -10221,6 +10221,14 @@
 			"icon": "xui.png",
 			"script": "[^a-z]xui.*\\.js",
 			"website": "http://xuijs.com"
+		},
+		"Moon": {
+			"cats": [
+				"12"
+			],
+			"icon": "moon.png",
+			"script": "moon(?:\\.min)?.js",
```
Comment (gadcam) [inline src/apps.json]: I agree with @jvoisin : the pattern should start with / and the final dot must be escaped. If the URL doesn't take any argument then you can even add $ at the end.
So the pattern could be <code>"script": "/moon(?:\\\\.min)?\\\\.js$"</code>

Comment (ctl) [inline src/apps.json]: thank you for help

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**